index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute page</title>
+    <link rel="stylesheet" type="text/css" href="stylesheet.css">
+</head>
+<body>
+    <div id = "main">
+        <h1 id = "title">Nelson Mandela</h1>
+    </div>
+    <div id="img-div">
+        <img id="image" src="https://bit.ly/3fKov2z" 
+        alt="Nelson Mandela Photo" class="responsive">
+        <p id = "img-caption">Nelson Mandela was a social rights activist, politician and philanthropist,
```

Comment:
I would recommend using the class selector for attributes instead of the id selector.